kuma/wiki/models.py

```diff
@@ -2012,3 +2012,28 @@ class DocumentSpamAttempt(SpamAttempt):
 
     def __unicode__(self):
         return u'%s (%s)' % (self.slug, self.title)
+
+
+class BCSignal(models.Model):
+    """Model to keep track of the BC signals."""
+    document = models.ForeignKey(
+        Document,
+        related_name='bc_signals',
+        null=True,
+        blank=True,
+        verbose_name=_('Document (optional)'),
+        on_delete=models.SET_NULL
+    )
+    user = models.ForeignKey(
```

Comment:
Have we thought carefully about whether we want to associate these signals with users? If we never collect the user id in the first place then we'll never have to worry about scrubbing user ids from the signal data before sharing it.

Given that most signals will come from non-logged in users, is there any real point in collecting the user data?